fire/cli/mtl.py

```diff
@@ -446,14 +420,31 @@ def opbyg_punktoversigt(
         if punkt in nye_punkter:
             continue
 
-        info = punkt_information(punkt)
-        kote = punkt_kote(info, koteid)
-        if kote is not None:
-            punktoversigt.at[punkt, "kote"] = kote.z
-            punktoversigt.at[punkt, "σ"] = kote.sz
-            punktoversigt.at[punkt, "år"] = kote.registreringfra.year
+        fire.cli.print(f"Finder kote for {punkt}", fg="green")
+        pkt = firedb.hent_punkt(punkt)
+
+        # Grav aktuel kote frem
+        kote = None
+        for koord in pkt.koordinater:
+            if koord.sridid != DVR90.sridid:
```

Comment:
Du burde kunne undlade ideerne her:
```suggestion
            if koord.srid != DVR90:
```

Reply:
Ja - det havde jeg også overvejet, men tænkte at der ikke var grund til at checke identitet af hele objektet, når det kunne afgøres på en enkelt attribut.

På den anden side er det ulighed vi checker, så spørgsmålet er afgjort så snart en enkelt bit afviger - og din version er mindre mentalt belastende ved at kunne læses rent hverdagssprogligt, så jeg tror jeg vælger den